packages/compass-indexes/src/components/index-header/index-header.module.less

```diff
@@ -1,21 +1,5 @@
 .index-header {
-  table tr th {
-    cursor: pointer;
-    user-select: none;
-    -webkit-user-select: none;
+  th {
+    padding: 12px 24px;
```

Comment:
Before:
<img width="1164" alt="Screen Shot 2022-02-16 at 7 11 45 PM" src="https://user-images.githubusercontent.com/1791149/154380262-723e5049-8e3b-4435-9440-84ed9925f2d0.png">

After:
<img width="1043" alt="Screen Shot 2022-02-16 at 7 22 18 PM" src="https://user-images.githubusercontent.com/1791149/154380253-91e3957e-7205-4b22-90cb-05461ddee799.png">


Also not sure when these styles broke, I have a version of 1.21 and they're looking alright in that.